<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BoardEditor</class>
 <widget class="QWidget" name="BoardEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>160</width>
    <height>281</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="barrenButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>0</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(192,192,192);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoRepeatDelay">
    <number>290</number>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">FieldsButtonsGroup</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="groundButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(200, 150, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoRepeat">
    <bool>false</bool>
   </property>
   <property name="autoExclusive">
    <bool>false</bool>
   </property>
   <property name="autoRepeatDelay">
    <number>300</number>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">FieldsButtonsGroup</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="waterButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>0</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoRepeatDelay">
    <number>290</number>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">FieldsButtonsGroup</string>
   </attribute>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>0</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>50</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-bottom-color: rgb(255, 0, 4);</string>
   </property>
   <property name="text">
    <string>X</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="autoRepeatDelay">
    <number>290</number>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">FieldsButtonsGroup</string>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="FieldsButtonsGroup">
   <property name="exclusive">
    <bool>true</bool>
   </property>
  </buttongroup>
 </buttongroups>
</ui>
